service/src/main/java/com/myhome/repositories/HouseHistoryRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.myhome.repositories;
+
+import com.myhome.domain.HouseHistory;
+
+import java.util.List;
+
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface HouseHistoryRepository extends PagingAndSortingRepository<HouseHistory,Long> {
+
+  List<HouseHistory> findByHouseId(String houseId);
```

Comment:
Please take a look at FIND vs GET naming convention:

https://softwareengineering.stackexchange.com/questions/182113/how-and-why-to-decide-between-naming-methods-with-get-and-find-prefixes/182120